h/models/moderation_log.py

```diff
@@ -10,17 +12,22 @@
 from h.db.mixins_dataclasses import AutoincrementingIntegerID, CreatedMixin
 from h.models.annotation import ModerationStatus
 
+if TYPE_CHECKING:
+    from h.models import Annotation, User
+
 
 class ModerationLog(Base, AutoincrementingIntegerID, CreatedMixin, MappedAsDataclass):
```

Comment:
Changes to this dataclass definition to be able to instanciate with a full `moderator` object (or None) as apposoed to just with the FK, `moderator_id`


See: https://hypothes-is.slack.com/archives/C1MA4E9B9/p1746092504055169